.env.example

```
@@ -1,53 +1,4 @@
-MATRIX_DATABASE_PASSWORD='synapse!1'
-
-SMS_API_KEY=SOME_SECRET_KEY
-SMS_API_LOGIN=login@domain.com
-SMS_API_URL=https://api.octopush.com/v1/public
-
-LOG_LEVEL=silly
-LOG_TRANSPORTS=Console
-
-FEATURE_COMMON_SETTINGS_ENABLED=false
+NODE_TLS_REJECT_UNAUTHORIZED=0
+NODE_EXTRA_CA_CERTS=.compose/ssl/certs/rootCA.pem
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_EXTRA_CA_CERTS key should go before the NODE_TLS_REJECT_UNAUTHORIZED key

(UnorderedKey)